docs/license.rst

```diff
@@ -14,7 +14,7 @@ these elements still apply and are not touched by the software license.
 ::
 
   EvaP - Evaluation Platform
-  Copyright (C) 2011-2014 by Michael Grünewald, Stefan Richter, Johannes Linke and Johannes Wolf
+  Copyright (C) 2011-2015 by Michael Grünewald, Stefan Richter, Johannes Linke and Johannes Wolf
 
```

Comment:
Don't worry, we'll clean it up. Next time please make sure to not commit existing changes again :)